torch_xla/csrc/tensor_impl.cpp

```diff
@@ -168,10 +172,31 @@ void XLATensorImpl::SetupSizeProperties() {
     for (int i = 0; i < updated_strides.size(); i++) {
```

Comment:
if I understand your point, you are suggesting to duplicate the strides logic inside `SetupSymSizeProperties` as well, so it updates both sym_sizes and sym_strides the same way `SetupSizeProperties`

If this is your concern, I believe it's fine for now, since `sym_strides` is still returning static strides now.
We should figure out if we need support for dynamic strides as well or whether it's okay for sym_strides to continue returning static strides.

Reply:
@ezyang: based on our offline conversation about `sym_strides` under functionalization, it seems we assume no need for dynamic strides; correct?